neurovault/apps/statmaps/views.py

```diff
@@ -375,7 +375,7 @@ def upload_folder(request, collection_cid):
                             dom = minidom.parse(os.path.join(root, fname))
                             for atlas in dom.getElementsByTagName("summaryimagefile"):
                                 print "found atlas"
-                                path, base, ext = split_filename(atlas.lastChild.nodeValue)
+                                path, base = os.path.split(atlas.lastChild.nodeValue)
```

Comment:
If that's the case this change should not make any difference...

On Wed Jan 07 2015 at 1:04:52 PM jbwexler <notifications@github.com> wrote:

> In neurovault/apps/statmaps/views.py
> <https://github.com/NeuroVault/NeuroVault/pull/98#discussion-diff-22615713>
> :
>
> > @@ -375,7 +375,7 @@ def upload_folder(request, collection_cid):
> >                              dom = minidom.parse(os.path.join(root, fname))
> >                              for atlas in dom.getElementsByTagName("summaryimagefile"):
> >                                  print "found atlas"
> > -                                path, base, ext = split_filename(atlas.lastChild.nodeValue)
> > +                                path, base = os.path.split(atlas.lastChild.nodeValue)
>
> actually atlas.lastChild.nodeValue doesn't have an extension. ex:
> '/HarvardOxford/HarvardOxford-cort-maxprob-thr25-2mm'
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/NeuroVault/NeuroVault/pull/98/files#r22615713>.
>